jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -352,10 +386,15 @@ object XGBoost extends Serializable {
   }
 
   private def postTrackerReturnProcessing(
-      trackerReturnVal: Int, distributedBoosters: RDD[Booster],
-      params: Map[String, Any], sparkJobThread: Thread, isClassificationTask: Boolean):
-    XGBoostModel = {
+      trackerReturnVal: Int,
+      distributedBoosters: RDD[Booster],
+      sparkJobThread: Thread,
+      isClassificationTask: Boolean): XGBoostModel = {
     if (trackerReturnVal == 0) {
+      /*
+      Copies of the finished model reside in each partition of the `distributedBoosters`.
+      Any of them can be used to create the model. Here, just choose the first partition.
+      */
```

Comment:
please change it to `//` style, which is consistent with other places

/* */ is usually used right above class/method signature ,